certbot/certbot
# Remove commas in filename

This will immediately address the breakage reported in #6682 and tracked at #6685. Pip 19.0.0 and 19.0.1 don't allow commas in filenames, so don't use commas in filenames in certbot-apache test code.

I've confirmed that this fixes the issue on a machine that fails with the version of certbot-auto currently in master: recent version of virtualenv, python 2.7.

Steps to test:
- push master to test box
- run tools/venv.py
- activate venv
- pip --version: 19.0.1
- pip install ./certbot-apache/: fails
- push branch code to test box
- confirm pip --version still 19.0.1
- pip install ./certbot-apache/: success
